src/bin/cairo-native-run.rs

```diff
@@ -268,16 +257,15 @@ fn jitvalue_to_felt(value: &JitValue) -> Vec<Felt> {
                 if debug_name == "core::bool" {
                     vec![(*tag == 1).into()]
                 } else {
-                    felts.push((*tag).into());
-                    let felt = jitvalue_to_felt(value);
-                    felts.extend(felt);
+                    let mut felts = vec![(*tag).into()];
+                    // felts.push((*tag).into());
```

Comment:
Why did you comment that line?

Reply:
Sorry, we can just remove it as the vec is initialized on the line before with the tag. Just fixed